gh-badges/lib/make-badge.js

```diff
@@ -109,12 +107,27 @@ function makeBadge({
   // String coercion and whitespace removal.
   text = text.map(value => `${value}`.trim())
 
-  if (format !== 'json') {
-    format = 'svg'
+  const [left, right] = text
+
+  color = normalizeColor(color || colorB || colorscheme)
+  labelColor = normalizeColor(labelColor || colorA)
+
+  // This ought to be the responsibility of the server, not `makeBadge`.
+  if (format === 'json') {
+    return JSON.stringify({
+      label: left,
+      message: right,
+      logoWidth,
+      color,
+      labelColor,
+      link: links,
```

Comment:
Is there a need to do any of the encoding `escapeXml` is doing for the json response? I can't think of anything off hand but figured I'd ask anyway

Reply:
Nope! Though now that you mention it, I think that is what's caused the (desirable) change in the link where `&` are no longer escaped as `&amp;`.